docs/development_suite/identity-and-access-management/console-levels-and-permission-management.md

```diff
@@ -30,58 +31,85 @@ Users and Service Accounts can perform a set of predetermined actions along Cons
-| Delete any Company                                                         | `console.root.company.delete`                          |        |          |           |            |                       |               |
-| Create a new Project                                                       | `console.root.project.create`                          |        |          |           |            |                       |               |
-| Edit any Project                                                           | `console.root.project.details.update`                  |        |          |           |            |                       |               |
-| Delete any Project                                                         | `console.root.project.delete`                          |        |          |           |            |                       |               |
-| View all Console resources                                                 | `console.root.view`                                    |        |          |           |            |                       |               |
-| Manage identity Roles, Groups and Bindings                                 | `console.root.user.bind`                               |        |          |           |            |                       |               |
-| Create and delete any user                                                 | `console.root.user.manage`                             |        |          |           |            |                       |               |
-| Manage all private and public Project Templates                            | `console.root.templates.manage`                        |        |          |           |            |                       |               |
-| Manage available features                                                  | `console.root.features.manage`                         |        |          |           |            |                       |               |
-| Manage Company Project Templates                                           | `console.company.templates.manage`                     |        |          |           |            |                       | ✅            |
 
-User Roles are manageable from CMS by the **Console administrators**, as they are the only ones having access to the Console CMS and thus being able to manage the entire Console, including Companies, Projects and the Marketplace.
+User Roles and permissions are manageable from CMS by **Console Super Users**, which are particular Console Administrators having access to the Console CMS and thus being able to perform actions at Console root level.
+
+:::info
+Note that Console Super User is NOT included as a default Role. Consequently, it can not be assigned from the Identities portal, as it performs actions exclusively at Console root level. 
```

Comment:
```suggestion
Note that Console Super User is NOT included as a default role. Consequently, it can not be assigned from the Identities portal, as it performs actions exclusively at Console root level. 
```